cypress/page/shoping-cart.page.ts

```diff
@@ -0,0 +1,21 @@
+class ShopingCartPage {
+    private ShopingCart: string;
+    private CartNavigation: string
+
+    constructor() {
+        this.ShopingCart = '[style*="display: block;"] .button-container > a';
+        this.CartNavigation = '.cart_navigation span'   
+    }
+    
+
+    public LookShopingCart(): void {
+        cy.get(this.ShopingCart).click()
+    }
+
+    public goToCartNavigation(): void {
```

Comment:
This is referring to click on the proceed to checkout button, let's name the method in a way that indicate that we are proceeding to checkout